sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizer.java

```diff
@@ -66,4 +67,8 @@ AutoConfigurationCustomizer addSpanExporterCustomizer(
    */
   AutoConfigurationCustomizer addPropertiesSupplier(
       Supplier<Map<String, String>> propertiesSupplier);
+
+  AutoConfigurationCustomizer addTracerProviderCustomizer(
+      BiFunction<SdkTracerProviderBuilder, ConfigProperties, SdkTracerProviderBuilder>
+          tracerProviderCustomizer);
```

Comment:
Need a default implementation for this since the `:sdk-extensions:autoconfigure-spi` module is stable.